bin/torii/src/main.rs

```diff
@@ -111,8 +112,12 @@ struct Args {
     events_chunk_size: u64,
 
     /// Enable indexing pending blocks
-    #[arg(long)]
+    #[arg(long, default_value_t = true)]
```

Comment:
```suggestion
    #[arg(long, action = ArgAction::Set, default_value_t = true)]
```
Without specifying the action, the boolean is not parseable as it's considered as a flag. Hence, impossible to set it to false.
With this, we can do:
```bash
torii --index-pending true
torii --index-pending false

# false
torii
```